src/you_get/extractors/bilibili.py

```diff
@@ -135,6 +136,15 @@ def test_bili_cdns(urls_list):
     return -1, None, 0
 
 def bilibili_download_by_cid(cid, title, output_dir='.', merge=True, info_only=False):
+        info_u = 'http://interface.bilibili.com/player?id=cid:' + cid
+        video_info = get_content(info_u).split('\n')
+        oriurl = ''
+        for line in video_info:
+            if line.startswith('<oriurl>'):
+                oriurl = line[len('<oriurl>'):-len('</oriurl>')]
+        if len(oriurl) != 0:
+            qq_download(oriurl, output_dir=output_dir, merge=merge, info_only=info_only)
```

Comment:
Inferred from legacy code in bilibili.py

In ```bilibili_download_by_cid``` some parameters are already processed and discarded. 
So kick it back will be messy. Lots of methods in ```common.py``` can act as an entrance so it will be hard to choose one.

There could be some videos using youku cloud player. We can fix it with adding several lines of code. That won't be hurt.